photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/training/Params.scala

```diff
@@ -216,7 +216,7 @@ object Params {
       opt[String]("output-dir")
         .required()
         .text(s"Output directory for logs and learned models.")
-        .foreach(x => params.outputDir = x.replace(',', '_').replace(':', '_'))
+        .foreach(x => params.outputDir = x.replace(',', '_'))
```

Comment:
We could make this a little more robust by having it parse URIs first, and then do the replacement -- if the intention is to support URIs as an input here (and if it's important to do the colon replacement).

I don't think we need to do this now though.